### PR TITLE
Add ClickHouse monitor e2e test

### PR DESCRIPTION
**Pull request by yanjunz97**
Signed-off-by: Yanjun Zhou <zhouya@vmware.com>

### CODE REVIEW

File: test/e2e/flowvisibility_test.go [hunk 1]
```diff
@@ -462,6 +464,90 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
...
+		}
+		checkClickHouseMonitor(t, data, isIPv6, flow)
+	})
+}
+
+func checkClickHouseMonitor(t *testing.T, data *TestData, isIPv6 bool, flow testFlow) {
+	checkClickHouseMonitorLogs(t, data, false, 0)
+	var cmdStr string
+	if !isIPv6 {
+		cmdStr = fmt.Sprintf("iperf3 -u -c %s -P 128 -n 1", flow.dstIP)
```
Comment (dreamtalen) [inline test/e2e/flowvisibility_test.go:496]: Better to have a comment here describing the choice of `128` as the max value for parallel clients.

**Review by dreamtalen — COMMENTED** [review]
Overall LGTM

File: hack/generate-manifest.sh [hunk 2]
```diff
@@ -23,9 +23,15 @@ function echoerr {
...
-        --mode (dev|release)  Choose the configuration variant that you need (default is 'dev')
-        --spark-operator      Generate a manifest with Spark Operator enabled.
-        --no-grafana          Generate a manifest with Grafana disabled.
+        --mode (dev|release)                Choose the configuration variant that you need (default is 'dev')
+        --spark-operator                    Generate a manifest with Spark Operator enabled.
+        --no-grafana                        Generate a manifest with Grafana disabled.
+        --ch-size <size>                    Deploy the ClickHouse with a specific storage size. Can be a 
+                                            plain integer or as a fixed-point number using one of these quantity
+                                            suffixes: E, P, T, G, M, K. Or the power-of-two equivalents:
+                                            Ei, Pi, Ti, Gi, Mi, Ki.  The default is 8Gi.
```
[the remark this answers is not in the document]
Comment (yanjunz97) [reply hack/generate-manifest.sh]: Removed space and updated the description a little bit. Thanks!

File: test/e2e/flowvisibility_test.go (hunk 1 above)
Comment (yanjunz97) [reply test/e2e/flowvisibility_test.go:497]: Added, thanks!

**Comment by yuntanghsu** [comment]
LGTM

**Review by dreamtalen — APPROVED** [review]
LGTM

**Comment by yanjunz97** [comment]
Squashed commits

**Pull request merged**